go.mod

```diff
@@ -10,6 +10,7 @@ retract (
 require (
 	cloud.google.com/go/storage v1.51.0
 	github.com/Masterminds/sprig/v3 v3.3.0
+	github.com/antithesishq/antithesis-sdk-go v0.5.1-0.20250924165633-f60b0222f1b6
```

Comment:
Don't they have a stable release? We shouldn't have unstable dependencies in the server codebase.

Reply:
No, they created this for just for us for now - I expect them to push a proper release once we report back.

Reply:
Keep in mind that this has absolutely zero effect (ie is dead code eliminated) as it's behind a build flagged source code and the linter prevents you from using it.

Reply:
Thinking about this more; I actually don't need to add the Antithesis-specific parts anymore if we extend `common/softassert/softassert.go` as described here. I can just swap out the implementation on my end. Let me do that.